ps_wirepayment.php

```diff
@@ -69,8 +77,11 @@ public function __construct()
             $this->reservation_days = $config['BANK_WIRE_RESERVATION_DAYS'];
         }
 
-        $this->bootstrap = true;
-        parent::__construct();
+        $config['BANK_WIRE_CUSTOM_TEXT'] = Configuration::get('BANK_WIRE_CUSTOM_TEXT', $this->context->language->id);
+        if (!empty($config['BANK_WIRE_CUSTOM_TEXT'])) {
+            $this->custom_text = $config['BANK_WIRE_CUSTOM_TEXT'];
```

Comment:
Why not use `$config['BANK_WIRE_CUSTOM_TEXT'];` ? Why add a 2nd variable `$this->custom_text`?